de.fu_berlin.inf.dpp.core/src/de/fu_berlin/inf/dpp/session/ISarosSession.java

```diff
@@ -361,97 +363,102 @@ public void addActivityConsumer(IActivityConsumer consumer,
-     *         session.
+     * @return a list of all shared resources (excluding reference points) from
+     *         this session.
      */
     public List<IResource> getSharedResources();
 
     /**
-     * Returns a map with the mapping of shared resources to their project.
+     * Returns a map with the mapping of shared resources to their reference
+     * point.
      * 
-     * @return project-->resource mapping
+     * @return reference point-->resource mapping
      */
-    public Map<IProject, List<IResource>> getProjectResourcesMapping();
+    public Map<IReferencePoint, List<IResource>> getReferencePointResourcesMapping();
 
     /**
-     * Returns the shared resources of the project in this session.
+     * Returns the shared resources of the reference point in this session.
      * 
-     * @param project
-     * @return the shared resources or <code>null</code> if this project is not
-     *         or fully shared.
+     * @param referencePoint
+     * @return the shared resources or <code>null</code> if this reference point
+     *         is not or fully shared.
      */
-    public List<IResource> getSharedResources(IProject project);
+    public List<IResource> getSharedResources(IReferencePoint referencePoint);
 
     /**
-     * Stores a bidirectional mapping between <code>project</code> and
-     * <code>projectID</code>.
+     * Stores a bidirectional mapping between <code>referencePoint</code> and
+     * <code>referencePointID</code>.
      * <p>
      * This information is necessary for receiving (unserializing)
      * resource-related activities.
      * 
-     * @param projectID
-     *            Session-wide ID of the project
-     * @param project
-     *            the local representation of the project
+     * @param referencePointID
+     *            Session-wide ID of the reference point
+     * @param referencePoint
+     *            the local representation of the reference point
      * 
-     * @see #removeProjectMapping(String, IProject)
+     * @see #removeReferencePointMapping(String, IReferencePoint)
      */
-    public void addProjectMapping(String projectID, IProject project);
+    public void addReferencePointMapping(String referencePointID,
+        IReferencePoint referencePoint);
 
     /**
-     * Removes the bidirectional mapping <code>project</code> and
-     * <code>projectId</code> that was created by
-     * {@link #addProjectMapping(String, IProject) addProjectMapping()} .
+     * Removes the bidirectional mapping <code>referencePoint</code> and
+     * <code>referencePointId</code> that was created by
+     * {@link #addReferencePointMapping(String, IReferencePoint)
+     * addReferencePointMapping()} .
      * <p>
-     * TODO Why is the project parameter needed here? This forces callers to
-     * store the mapping themselves (or retrieve it just before calling this
+     * TODO Why is the referencePoint parameter needed here? This forces callers
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Complete the task associated to this TODO comment. ([squid:S1135](https://sonarcloud.io/organizations/saros/rules#rule_key=squid%3AS1135))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=saros_complete&pullRequest=210&issues=AWVWjH3VlRPS_ea-bThO&open=AWVWjH3VlRPS_ea-bThO)

[ISSUE_KEY]: AWVWjH3VlRPS_ea-bThO